riscv/include/mach_acpi.h

```diff
@@ -0,0 +1,27 @@
+/* acpi.h */
```

Comment:
first reaction was, oh no, we won't do ACPI in riscv. Second reaction was, oh, well, thanks to linux we probably will :-(

But why are we building acpi stuff for riscv right now? I never needed to. Or was I doing it without knowing it?